cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -406,6 +406,11 @@ func StartControllers(s *options.CMServer, kubeClient *client.Client, kubeconfig
 	volumeController.Run()
 	time.Sleep(wait.Jitter(s.ControllerStartInterval.Duration, ControllerStartJitter))
 
+	eventBroadcaster := record.NewBroadcaster()
```

Comment:
There is an `eventBroadcaster`/`recorder` constructed and used above in the `Run(...)` method. I'd suggest reusing that instead of creating a new one here. That will require passing an extra `recorder` parameter to `StartControllers(...)`